README.md

```diff
@@ -1,2 +1,28 @@
 # dune-adapter
 **experimental** preCICE-adapter for DUNE, a modular toolbox for solving partial differential equations
+
+A adapter module enabeling multiphysics simulation inside DUNE with the help of preCICE [1][2].
+
+The dune-adapter module depends on the DUNE core modules and
+on dune-elastodynamics providing the necessary tools for structural simulation.
+
+The dune-elastodynamics module can be found here:
+https://github.com/maxfirmbach/dune-elastodynamics
+
+The adapter can be build with:
+`<path-to-dune-common/bin/dunecontrol> --current all`
```

Comment:
I installed dune from apt, where `dunecontrol` is in the global searchpath:
```suggestion
`dunecontrol --current all`
```

Reply:
Maybe link here also to the module installation instructions provided by DUNE itself. Is the `dunecontrol make install` command missing?